client/components/auth/auth.service.js

```diff
@@ -59,9 +59,7 @@ function AuthService($location, $http, $cookies, $q, appConfig, Util, User) {
     createUser(user, callback) {
       return User.save(user,
         function(data) {
-          $cookies.put('token', data.token);
-          currentUser = User.get();
-          return safeCb(callback)(null, user);
+          $location.path('/verify/emailVerification');
```

Comment:
This check should also happen at login. There is nothing stopping them from just logging in or anything prompting them to verify email